### PR TITLE
ci: Update labeler.yml to V5 syntax

### PR DESCRIPTION
**Pull request by teolemon**
### What
- ci: Update labeler.yml to V5 syntax

### CODE REVIEW

**Comment by raphodn** [comment]
closing (reverted #735 in https://github.com/openfoodfacts/open-prices-frontend/pull/763)

**Comment by raphodn** [comment]
reopening after re-reverting #765

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,42 +1,54 @@
...
-- .github/**/*
+- changed-files:
+  - any-glob-to-any-file: '.github/**/*'
 
 Price addition:
-- 'src/views/AddPriceHome.vue'
-- 'src/views/AddPriceSingle.vue'
-- 'src/views/AddPriceMultiple.vue'
+- changed-files:
+  - any-glob-to-any-file: 'src/views/AddPriceHome.vue'
```
Comment (raphodn) [inline .github/labeler.yml:9]: shouldn't it be 1 line with an array of values ?

https://github.com/actions/labeler#basic-examples

Comment (teolemon) [reply .github/labeler.yml:9]: it works with my syntax on other repos 🤷

**Comment by raphodn** [comment]
@teolemon doesn't seem to work ? see one of the latest PRs (for instance #769)

**Comment by teolemon** [comment]
- instead of reverting, let's fix master so that the labeler works again. The "actions" tab is there for us
- also for release please, you're using slightly different conventions from the rest of the repos, so I need to have an extra look (but I think we should just fix the config in crowdin or release please, and missing a PR or 2 is not dramatic)

**Pull request merged**